src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipEntryProcessor.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.tasks.processor;
+
+import org.apache.commons.io.FilenameUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.hmcts.reform.bulkscanprocessor.exceptions.NoPdfFileFoundException;
+import uk.gov.hmcts.reform.bulkscanprocessor.services.document.output.Pdf;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+
+import static com.google.common.io.ByteStreams.toByteArray;
+
+public class ZipEntryProcessor {
```

Comment:
I'd say ZipEntriesProcessor or ZipFileProcessor. This does not process a single entry as far as I can see.